build.make

```diff
@@ -63,22 +63,22 @@ endif
 # Specific packages can be excluded from each of the tests below by setting the *_FILTER_CMD variables
 # to something like "| grep -v 'github.com/kubernetes-csi/project/pkg/foobar'". See usage below.
 
-# BUILD_PLATFORMS contains a set of <os> <arch> <suffix> triplets,
+# BUILD_PLATFORMS contains the tuple [os arch suffix base_image],
```

Comment:
I find this confusing. It contains not just one tuple.

Reply:
fixed